lib/manageiq-api.rb

```diff
@@ -1,2 +1,25 @@
-require "manageiq/api/engine"
-require "manageiq/api/version"
+
+require "zeitwerk"
+loader = Zeitwerk::Loader.for_gem(warn_on_extra_files: false)
+loader.ignore(__FILE__)
+
+# Add non-standard root directory to the path
+loader.push_dir("#{__dir__}/../app/controllers")
```

Comment:
I would expect any rails engine to have this (as well as things like models, etc - do we have to do this in every plugin?

Reply:
I'll look.  Thanks.